lib/data/landing_page_content_items/tasks.yaml

```diff
@@ -149,6 +149,23 @@ blocks:
             csv_file: "data_one.csv"
             data_source_link: https://www.ons.gov.uk/economy/grossdomesticproductgdp/timeseries/ihyq/qna
             padding: true
+- type: two_column_layout
+  theme: two_thirds_one_third
+  blocks:
+  - type: press_notices
+    items:
+    - text: An example title for this press notice 1
+      path: https://www.gov.uk
+      document_type: Press release
+      public_updated_at: "2024-10-27 10:29:44 +0000"
+    - text: An example title for this press notice 2
+      path: https://www.gov.uk
+      document_type: Press release
+      public_updated_at: "2024-10-16 11:34:12 +0000"
+    - text: An example title for this press notice 3
+      path: https://www.gov.uk
```

Comment:
Nitpick: can we use `href` instead of `path`? Just to make it consistent with (hopefully all of) the other blocks.

Reply:
Sure, although this one's using `document_list` under the hood which uses `path` instead. Should that one also change to `href`?

Reply:
You mean the component itself? No, that's fine, just here in the YML.

Reply:
OK this is is done.